backend/dining/migrations/0003_venue_name.py

```diff
@@ -11,6 +11,8 @@ class Migration(migrations.Migration):
 
     operations = [
         migrations.AddField(
-            model_name="venue", name="name", field=models.CharField(max_length=255, null=True),
+            model_name="venue",
+            name="name",
+            field=models.CharField(max_length=255, null=True),
```

Comment:
new black splits into lines if we have trailing commas, otherwise tries to one line. 
old black tried to one line and added trailing commas. 

To make things look similar for most files where we had one line previously I deleted the trailing comma and reformatted but I ignored for migration files.